retriever/lib/f_datapackage.py

```diff
@@ -0,0 +1,124 @@
+import os.path
+import yaml
+try:
+    from datapackage import Package 
+except ImportError as e:
+    print("Could not import Package - ",e) 
+from collections import OrderedDict
+from retriever.lib.templates import TEMPLATES
+from retriever.lib.models import myTables
+
+from retriever.lib.defaults import SCRIPT_SEARCH_PATHS
+
+
+"""  Uses url in datapackage.yml to load resources, then downloads the datapackage using datapackage-py
+     https://github.com/frictionlessdata/datapackage-py
+     cleans the json file with the required changes, then creates script module 
+"""
+
+def get_dps():
+    """Load the list of data packages from datapackages.yml
+    Checks all of the search paths for a datapackages.yml file and loads it into
+    a dictionary.
+    """
+    for path in SCRIPT_SEARCH_PATHS:
+        try:
```

Comment:
When you do a search for script(`SCRIPT_SEARCH_PATHS`). we start in the working directory, then `./scripts` until we get to the `~/.retriever/script directory`

As soon as we get the first script, in this case we look for `datapakages.yml`.  We should consider that as the most current file and jump out of the loop.

Reply:
I'm still trying to connect the dots here, this is taking forever..

What I understand is that `datapackages.yml` doesn't exit in `~/.retriever/scripts` ,
then I should probably try to solve that first.. yes?

I see that `SCRIPT_WRITE_PATH` points to `~/.retriever/scripts` and the function
`check_for_updates(repo=REPOSITORY)`  is being used to download the latest scripts to that path.

My thinking is I should create a similar function (or edit `check_for_updates`) for updating `datapackages.yml`. 
@henrykironde What are your views?